common.json

```diff
@@ -114,5 +114,11 @@
             "module": "codecoverage-tools",
             "dest": "node_modules"  
         }
+    ],
+    "XamarinAndroid": [
```

Comment:
Needed only in M105 (gulp build) to copy the ps_module to the task under the _build folder